modules/template/variables.tf

```diff
@@ -1,6 +1,6 @@
 variable "bucket" {
   description = "The S3 bucket used to store the template file."
-  type        = "string"
```

Comment:
Can you please rebase your branch? All of the changes in this file were already done in master a couple of days ago.